Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json

```diff
@@ -0,0 +1,563 @@
+            {
+                "name": " Ravi Kapur",
+                "orcid": "",
+                "affiliation": " Center for Engineering in Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Sridhar Ramaswamy",
+                "orcid": "https://orcid.org/0000-0001-6715-4686",
+                "affiliation": " Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": " Toshi Shioda",
+                "orcid": "",
+                "affiliation": "Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Mehmet Toner",
+                "orcid": "",
+                "affiliation": "Center for Engineering in Medicine, Harvard Medical School, Boston, MA 02114, USA; Department of Surgery, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Shyamala Maheswaran",
+                "orcid": "https://orcid.org/0000-0002-9356-1709",
+                "affiliation": " Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Surgery, Harvard Medical School, Boston, MA 02114, USA. Electronic address: maheswaran@helix.mgh.harvard.edu.",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Daniel A. Haber",
+                "orcid": "https://orcid.org/0000-0003-1378-6551",
+                "affiliation": "Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA; Howard Hughes Medical Institute, Chevy Chase, MD 20815, USA. Electronic address: haber@helix.mgh.harvard.edu.",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            }
+        ]
+    },
+    "usability_domain": [
+        "To identify the expression of extracellular matrix genes in mouse pancreatic CTCs and human CTCs originating from the pancreas, breasts, and prostate.",
+        "PMID: 25242334 PMCID: PMC4230325 DOI: 10.1016/j.celrep.2014.08.029",
```

Comment:
Great job including the publication. I would just add some context to make this a complete sentence : "This BCO was adapted by [title][PMID/PMCID/DOI]